modules/gltf/src/lib/gltf-utils/get-typed-array.ts

```diff
@@ -24,18 +27,54 @@ export function getTypedArrayForImageData(json, buffers, imageIndex) {
   return getTypedArrayForBufferView(json, buffers, bufferViewIndex);
 }
 
-/*
-// accepts accessor index or accessor object
-// returns a typed array with type that matches the types
-export function getTypedArrayForAccessor(accessor) {
-  accessor = this.getAccessor(accessor);
-  const bufferView = this.getBufferView(accessor.bufferView);
-  const buffer = this.getBuffer(bufferView.buffer);
-  const arrayBuffer = buffer.data;
-
-  // Create a new typed array as a view into the combined buffer
-  const {ArrayType, length} = getAccessorArrayTypeAndLength(accessor, bufferView);
-  const byteOffset = bufferView.byteOffset + accessor.byteOffset;
-  return new ArrayType(arrayBuffer, byteOffset, length);
+/**
+ * Gets data pointed by the accessor.
+ * @param json - json part of gltf content of a GLTF tile.
+ * @param buffers - Array containing buffers of data.
+ * @param accessor - accepts accessor index or accessor object.
+ * @returns {TypedArray} Typed array with type matching the type of data poited by the accessor.
+ */
+// eslint-disable-next-line complexity
+export function getTypedArrayForAccessor(
+  json: GLTF,
+  buffers: GLTFExternalBuffer[],
+  accessor: GLTFAccessor | number
+): TypedArray {
+  const gltfAccessor = typeof accessor === 'number' ? json.accessors?.[accessor] : accessor;
+  if (!gltfAccessor) {
+    throw new Error(`No Accessor ${accessor}`);
```

Comment:
Can be very hard for end users to understand such an error message. Maybe mentioning glTF could give them a hint?
```suggestion
    throw new Error(`No glTF accessor ${accessor}`);
```